server/selective_sync.go

```diff
@@ -23,13 +22,12 @@ func (p *Plugin) ChatShouldSync(channelID string) (bool, *model.AppError) {
 }
 
 // ChatMembersSpanPlatforms determines if the given channel members span both Mattermost and
-// MS Teams. Chats between users on the same platform are skipped if selective sync is enabled.
+// MS Teams. Only Synthetic users are considered MS Teams users.
+// Chats between users on the same platform are skipped if selective sync is enabled.
```

Comment:
The last part of this comment is probably ripe for removal, given that this function is used for other purposes now. Propose:
```suggestion
// ChatMembersSpanPlatforms determines if the given channel members span both Mattermost and
// MS Teams. Synthetic users are considered to be on MS Teams, and all others on Mattermost.
```